index.js

```diff
@@ -28,10 +28,12 @@ const getAsString = (string, options) => string.replace(groupedIssueRegex, match
 
 const getAsDocumentFragment = (string, options) => {
 	const fragment = document.createDocumentFragment();
-	for (const [index, text] of Object.entries(string.split(groupedIssueRegex))) {
-		if (index % 2) { // URLs are always in odd positions
+	const parts = string.split(groupedIssueRegex);
+
+	for (const [index, text] of parts.entries()) {
+		if (index % 5 === 1) { // Issues are always first in capturing group
```

Comment:
MAGIC_NUMBER_ALERT

`issue-regex` must be use a fixed version in order to guarantee this. Next thing you know someone alters the group count in the package and this fails again.

There should also be a `const issueRegexGroups = 5;` constant

Reply:
> Next thing you know

…a PR for this is submitted:

- https://github.com/sindresorhus/issue-regex/issues/15

Reply:
[`8325460` (#13) (L5)](https://github.com/sindresorhus/linkify-issues/pull/13/commits/8325460e646bc3b94cc093e955dd01c8a4552fd8#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R5) returns the group count

Reply:
Smart! 😂 no need to pin the version then

Reply:
I would still actually pin the version, in case the order of groups changes

Reply:
I don’t think that changes since the first group is what we create on line 4, there can't be an earlier group